src/main/java/com/mraof/minestuck/client/util/MSKeyHandler.java

```diff
@@ -91,15 +92,27 @@ public static void onKeyInput(InputEvent event)	//This is only called during the
 	@SubscribeEvent
 	public static void onTick(TickEvent.ClientTickEvent event)
 	{
+
 		if(InputMappings.isKeyDown(Minecraft.getInstance().getMainWindow().getHandle(), captchaKey.getKey().getKeyCode()) && !captchaKeyPressed) {
 
 			Screen screen = Minecraft.getInstance().currentScreen;
-			if(screen instanceof ContainerScreen<?> && screen.getFocused() == null && !(screen instanceof CreativeScreen))
+
+			if(screen instanceof ContainerScreen<?> && ((ContainerScreen<?>) screen).getSlotUnderMouse() != null && !(screen instanceof CreativeScreen))
 			{
 				Slot slot = ((ContainerScreen<?>) screen).getSlotUnderMouse();
+				Minecraft.getInstance().player.sendMessage(new StringTextComponent("Screen class name: " + screen.getClass().getSimpleName()));
+				Minecraft.getInstance().player.sendMessage(new StringTextComponent("slot index: " + slot.slotNumber));
 				if(slot != null)
 					MSPacketHandler.sendToServer(CaptchaDeckPacket.captchalogueInv(slot.slotNumber, ((ContainerScreen<?>) screen).getContainer().windowId));
 			}
+			else if(screen instanceof ContainerScreen<?> && ((ContainerScreen<?>) screen).getSlotUnderMouse() != null && (screen instanceof CreativeScreen))
```

Comment:
I'm gonna have a closer look at the creative screen case to make sure it's a reliable solution, and not just a solution which happen to work.